src/components/track-changes/utils.ts

```diff
@@ -38,7 +38,7 @@ export const filterNodesWithTrackingData = (node: any) => {
 }
 
 const getLastChange = (changes: TrackedAttrs[]) => {
-  return [...changes].sort((a, b) => (a.createdAt < b.createdAt ? 1 : 0))[0]
+  return [...changes].sort((a, b) => b.createdAt - a.createdAt)[0]
```

Comment:
should we use `updatedAt`. I can see in quarterback for some cases we just update `updatedAt`

Reply:
updated